src/main/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheck.java

```diff
@@ -179,15 +179,7 @@
 
     /** Regular expression for all escaped chars. */
     private static final Pattern ALL_ESCAPED_CHARS = Pattern.compile("^((\\\\u)[a-fA-F0-9]{4}"
-            + "|\""
-            + "|'"
-            + "|\\\\"
-            + "|\\\\b"
-            + "|\\\\f"
-            + "|\\\\n"
-            + "|\\\\r"
-            + "|\\\\t"
```

Comment:
not related to this issue but related to this check: [JEP 368](https://openjdk.java.net/jeps/368) adds a new escape to java: `\s`.

Reply:
@pbludov , bingo ! .... we missed it in grammar support.

Reply:
I postponing fix for Check till https://github.com/checkstyle/checkstyle/issues/9133

Reply:
https://github.com/checkstyle/checkstyle/pull/9134